src/domain/session/room/timeline/tiles/index.ts

```diff
@@ -28,17 +28,17 @@ import {EncryptionEnabledTile} from "./EncryptionEnabledTile.js";
 import {MissingAttachmentTile} from "./MissingAttachmentTile.js";
 
 import type {SimpleTile} from "./SimpleTile.js";
-import type {Room} from "../../../../../matrix/room/Room";
 import type {Timeline} from "../../../../../matrix/room/timeline/Timeline";
 import type {FragmentBoundaryEntry} from "../../../../../matrix/room/timeline/entries/FragmentBoundaryEntry";
 import type {EventEntry} from "../../../../../matrix/room/timeline/entries/EventEntry";
 import type {PendingEventEntry} from "../../../../../matrix/room/timeline/entries/PendingEventEntry";
 import type {Options as ViewModelOptions} from "../../../../ViewModel";
+import type {RoomViewModel} from "../../RoomViewModel";
 
 export type TimelineEntry = FragmentBoundaryEntry | EventEntry | PendingEventEntry;
 export type TileClassForEntryFn = (entry: TimelineEntry) => TileConstructor | undefined;
 export type Options = ViewModelOptions & {
-    room: Room,
+    roomVM: RoomViewModel,
```

Comment:
It's a little difficult to confirm due to current lack of typescript, but I'm pretty sure that this is what's meant. I only see `options.roomVM` used in `tiles/`:

```
$ grep -rn  "options\.room" src/domain/session/room/timeline/tiles
src/domain/session/room/timeline/tiles/SimpleTile.js:148:        return this._options.roomVM;
```

We also are already using it in this manner here:

https://github.com/vector-im/hydrogen-web/blob/master/src/domain/session/room/RoomViewModel.js#L53-L57